docs/data/talks.yaml

```diff
@@ -3,7 +3,7 @@ mercedes:
   presenter: Dr. Christian Wege, Dr. David Schumm
   affiliation: Mercedes-Benz Group AG
   time: "11:30"
-  room: Sizi 5
+  room: S 5
```

Comment:
Please don't change the meeting room designation.

This conflicts with the information on the [venue page](https://boschglobal.github.io/embedded-linux/venue/#floor-plan), which uses the definition "Sizi". This also is the room designation, that is used in eRooms.